packages/paste-codemods/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@twilio-paste/codemods",
+  "version": "0.0.1",
+  "status": "beta",
+  "description": "A collection of codemods for maintaining projects built with Paste.",
+  "author": "Twilio Inc.",
+  "license": "MIT",
+  "publishConfig": {
+    "access": "public"
+  },
+  "bin": "./bin/paste-codemod.js",
+  "scripts": {
+    "build": "node ./tools/create-package-mappings.js",
+    "test": "jest"
+  },
+  "dependencies": {
+    "chalk": "^2.4.2",
+    "execa": "^3.2.0",
+    "globby": "^10.0.1",
+    "inquirer": "^7.0.0",
+    "is-git-clean": "^1.1.0",
+    "jscodeshift": "^0.11.0",
+    "meow": "^5.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.11.4",
+    "@babel/plugin-proposal-object-rest-spread": "^7.8.3",
+    "@babel/preset-env": "^7.8.4",
+    "@twilio-paste/core": "*"
```

Comment:
Added so that lerna builds this package last and so that the mappings get generated correctly